modules/Collections/views/entry.php

```diff
@@ -34,7 +34,7 @@
 
     <h3 class="uk-flex uk-flex-middle uk-text-bold">
         <img class="uk-margin-small-right" src="@url($collection['icon'] ? 'assets:app/media/icons/'.$collection['icon']:'collections:icon.svg')" width="25" alt="icon">
-        { entry._id ? 'Edit':'Add' } @lang('Entry')
+        @lang(entry._id ? 'Edit Entry':'Add Entry')
```

Comment:
Sorry, this PR mixing up PHP  with JS. Correct way would be:

`{ entry._id ? '@lang("Edit Entry")':'@lang("Add Entry")' }`
or
`{ App.i18n.get(entry._id ? 'Edit Entry':'Add Entry') }`

and please create the PR for the _next_ branch. Thanks!